glide.yaml

```diff
@@ -4,6 +4,8 @@ excludeDirs:
 - cmd/service-catalog
 - pkg/build/builder
 import:
+- package: github.com/ugorji/go
```

Comment:
you're in this file and its an easy fix.  Update the branches for openshfit/api and openshfit/client-go please

Reply:
not now... can look at pinning when I do bumps for the 1.8.4 rebase